workflow-ui/core-android/src/main/java/com/squareup/workflow1/ui/container/OverlayDialogFactoryFinder.kt

```diff
@@ -8,42 +8,8 @@ import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
 
 /**
  * [ViewEnvironment] service object used by [Overlay.toDialogFactory] to find the right
- * [OverlayDialogFactoryScreenViewFactory]. The default implementation makes [AndroidOverlay]
+ * [OverlayDialogFactory]. The default implementation makes [AndroidOverlay]
  * work, and provides default bindings for [AlertOverlay] and [FullScreenOverlay].
- *
- * Here is how this hook could be used to provide a custom dialog to handle [FullScreenOverlay]:
```

Comment:
This example is no longer applicable because of back press handling?

Reply:
It was already inaccurate, and I'm kind of exhausted, TBH.